Gems/Atom/Tools/AtomToolsFramework/Code/Include/AtomToolsFramework/Window/AtomToolsMainWindowRequestBus.h

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+#pragma warning(disable : 4100)
+#include <AzCore/EBus/EBus.h>
+#include <AzCore/std/containers/vector.h>
+#include <AzCore/std/string/string.h>
+
+class QWidget;
+
+namespace AtomToolsFramework
+{
+    //! AtomToolsMainWindowRequestBus provides
+    class AtomToolsMainWindowRequests : public AZ::EBusTraits
```

Comment:
Looks like this is a singleton (and unused so far) - we can make this an `AZ::Interface` instead. We just would get rid of the `: public AZ::EBusTraits` and the other EBus info and instead do `AZ::Interface<AtomToolsMainWindowRequests>::Register(this);` in the main window constructor and `AZ::Interface<AtomToolsMainWindowRequests>::Unregister(this);` in the destructor instead of the bus connect/disconnect.

Interfaces are a little more lightweight and easy to use than EBuses, so we're pushing for their use where possible.

Reply:
I agree.  However, I would postpone that for a subsequent change to keep this focused on centralizing and cleaning common code.  This change will still require updating tests/scripts to use the new buses.  There are a handful of other buses that could also use conversions interfaces after this.

Reply:
I didn't realize this was replacing another bus - totally OK keeping it as a bus for now if that's the case, let's just make sure this is wired up correctly - we can an interface conversion/supplement later